src/lang/std/artifactGraph.ts

```diff
@@ -92,7 +92,13 @@ interface SegmentArtifactRich extends BaseArtifact {
 
 interface SweepArtifactRich extends BaseArtifact {
   type: 'sweep'
-  subType: 'extrusion' | 'revolve' | 'revolveAboutEdge' | 'loft' | 'sweep'
+  subType:
+    | 'extrusion'
+    | 'revolve'
+    | 'revolveAboutEdge'
+    | 'loft'
+    | 'sweep'
+    | 'extrusionTwist'
```

Comment:
Can we change the type of `subType` to `SweepSubType` imported from the ts-rs bindings so that the tsc error doesn't happen again?